debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java

```diff
@@ -308,6 +310,14 @@ public static BinaryHandlingMode parse(String value, String defaultValue) {
             .withDescription("The maximum number of records that should be loaded into memory while performing a snapshot")
             .withValidation(Field::isNonNegativeInteger);
 
+    public static final Field SNAPSHOT_MODE_TABLES = Field.create("snapshot.table")
```

Comment:
This could need some streamlining with existing option names and structures. How about "table.snapshot.include.list" as a name? And making it a list of regexps, as is done for the other similar options?

Reply:
Hi @gunnarmorling, apologies, if I have missed something, in order to make a `Field` a list of regexps, would the following suffice?

```
public static final Field SNAPSHOT_MODE_TABLES = Field.create("snapshot.include.collection.list")
            .withDisplayName("Snapshot Mode include Data Collection")
            .withType(Type.LIST)
            .withWidth(Width.LONG)
            .withImportance(Importance.MEDIUM)
            .withValidation(Field::isListOfRegex)
            .withDescription(
                    "this setting must be set to specify a list of tables/collections 
whose snapshot must be taken on creating or restarting the connector.");
```

Reply:
Yes, that's good for the definition. See other options (e.g. "table.include.list") for how this is handled then (splitting, evaluating this as regexp, etc.).